src/utils/isStandardSyntaxUrl.js

```diff
@@ -1,21 +1,42 @@
-import { hasInterpolation } from "../utils"
+import {
+  hasLessInterpolation,
+  hasPsvInterpolation,
+  hasScssInterpolation,
+} from "../utils"
+
 /**
  * Check whether a URL is standard
  *
  * @param {string} url
  * @return {boolean} If `true`, the url is standard
  */
 export default function (url) {
-  // SCSS or Less interpolation
-  if (hasInterpolation(url)) { return false }
+  if (url.length === 0) { return true }
+
+  // Sass interpolation works anywhere
+  if (hasScssInterpolation(url) || hasPsvInterpolation(url)) { return false }
 
-  // Inside `'` and `"` work only interpolation
+  // Inside `'` and `"` work only LESS interpolation
   if (url[0] === "'" && url[url.length - 1] === "'"
```

Comment:
@davidtheclark 
`&&` - have priority `6`
`||` - have priority `5`
First - all `&&` then `||`
Ref: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

But i add parens for best reading code.